integration_tests/TestdeBuscador.js

```diff
@@ -0,0 +1,19 @@
+const { Builder, By, until } = require("selenium-webdriver");
+const driver = new Builder().forBrowser("firefox").build();
+
+async function TestdeBusqueda()
+
+     {
+
+     await driver.get("http://127.0.0.1:5500/index.html"); /*este link tiene que ser modificado dependiendo del repositorio local de cada uno*/
```

Comment:
Agregar formato al test (tabulaciones, enters).  Si usas visual studio code, tiene un atajo para formatear el codigo (https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode).  Si usás otro ide, probablemente tenga algo similar.